app/components/containers/containersController.js

```diff
@@ -128,7 +130,11 @@ angular.module('containers', [])
           });
         }
         else if (action === Container.remove) {
-          action({id: c.Id}, function (d) {
+          var force = false;
+          if (c.State === 'running') {
```

Comment:
What's the point of enabling the `force` parameter when the container is running only? IMO, it should always be equal to `true`. Force removing a stopped container is equivalent to removing a stopped container. It will only take effect when removing a running container and the modal will ask for confirmation.